client/web/compose/src/components/PageBlocks/TabsConfigurator.vue

```diff
@@ -282,7 +282,6 @@ export default {
         fillJustify: [
           { text: this.$t('tabs.style.fillJustify.fill'), value: 'fill' },
```

Comment:
Remove fill or justify, not none. 
This also needs to be done in navigation configurator.